client/src/screens/Client/Manage/DeletePatient/DeletePatient.js

```diff
@@ -10,10 +10,10 @@ import Typography from "@material-ui/core/Typography";
 const useStyles = makeStyles((theme) => ({
   root: {
     flexGrow: 1,
-    padding: "40px 0px"
+    padding: "25px 0px"
   },
   title: {
-    paddingBottom: theme.spacing(1)
+    paddingBottom: theme.spacing(.5)
```

Comment:
Let use `theme.spacing(1/2)`

Reply:
can i add this changes with my next pr? because i have to change more than 10 file to resolve this

Reply:
It's easy to do from your editor. Just search for `theme.spacing(.5)`. 
Like: 
<img width="565" alt="Screen Shot 2020-10-19 at 15 15 41" src="https://user-images.githubusercontent.com/1988935/96455868-11177180-121e-11eb-99ce-3677efcc54c7.png">

Reply:
https://code.visualstudio.com/docs/editor/codebasics#:~:text=Find%20and%20Replace%23,editor%2C%20overview%20ruler%20and%20minimap.